applications/dashboard/controllers/class.usercontroller.php

```diff
@@ -106,15 +106,20 @@ public function index($Keywords = '', $Page = '', $Order = '') {
         }
 
         // Get user list
-        $this->UserData = $UserModel->Search($Filter, $Order, $OrderDir, $Limit, $Offset);
+        $this->UserData = $UserModel->search($Filter, $Order, $OrderDir, $Limit, $Offset);
         $this->setData('Users', $this->UserData);
-        if ($this->PastUserThreshold()) {
-            $this->setData('_CurrentRecords', $this->UserData->count());
+        if ($this->pastUserThreshold()) {
+            if ($count = $this->UserData->count()) {
+                $this->setData('_CurrentRecords', $count);
+            } else {
+                // No users have been searched for, so give the total users overall.
+                $this->setData('RecordCount', $UserModel->getCount());
```

Comment:
This should really be shown if there is no count AND no filter. Otherwise it will look a bit odd.